.github/workflows/ci.yml

```diff
@@ -25,6 +25,15 @@ jobs:
         run: |
           docker build -t buildenv - < docker-build-env/Dockerfile
 
+      - name: "Check code format"
+        run: |
+          clang-format --version
```

Comment:
are you sure that you want to run this outside of the container?
Is clang-format available there?